docs/src/javascript.md

```diff
@@ -0,0 +1,56 @@
+# Javascript API Reference
+
+This section contains the API reference for LanceDB Javascript API.
+
+## Installation
+
+```bash
+npm install vectordb
+```
+
+This will download the appropriate native library for your platform. We currently
+support:
+
+* Linux (x86_64 and aarch64)
+* MacOS (Intel and ARM/M1/M2)
+* Windows (x86_64 only)
+
+We do not yet support musl-based Linux (such as Alpine Linux) or arch64 Windows.
+
+## Usage
+
+### Basic Example
+
+```javascript
+const lancedb = require('vectordb');
+//connect to a local database
+const db = await lancedb.connect('data/sample-lancedb');
+// connect to LanceDB Cloud
+// const db = await lancedb.connect({
```

Comment:
1. it looks weird having this in comments
2. there's no context about LanceDB Cloud at all, or explaining what `your-project-slug` means, etc.